Feijia Chen HW1

fc2440 opened the pull request.

fc2440 committed:
Initial Commit

malecki commented:
Please remove the files that are not in the `hw1/` folder (and put both the Rmd and html files in the right place)

malecki commented:
```
git rm  2014-9-18-Feijia-Chen.Rmd
git rm  2014-9-18-Feijia-Chen.html
git add hw1/_posts/2014-9-18-Feijia-Chen.html
git commit -m "cleanup"
git push origin
```

fc2440 committed:
Fifth Commit

fc2440 committed:
Initial Commit

fc2440 committed:
Fifth Commit

fc2440 committed:
Fifth Commit

fc2440 committed:
Initial Commit

fc2440 committed:
Fifth Commit

fc2440 committed:
Fifth Commit

Merge branch 'gh-pages' of https://github.com/fc2440/qmssviz into gh-pages

Conflicts:
	hw1/_posts/2014-9-18-Feijia-Chen.html

fc2440 committed:
Fifth Commit

fc2440 committed:
Initial Commit
